See AWESOME_C_FLAGS in CMakeLists.txt when doing the next major verbump after 3.5.x.

Gentoo bug #509658.

diff --git a/awesomeConfig.cmake b/awesomeConfig.cmake
index 009be17..1c04d3c 100644
--- a/awesomeConfig.cmake
+++ b/awesomeConfig.cmake
@@ -16,7 +16,7 @@ option(COMPRESS_MANPAGES "compress manpages" ON)
 option(GENERATE_DOC "generate API documentation" ON)

 # {{{ CFLAGS
-add_definitions(-O1 -std=gnu99 -ggdb3 -rdynamic -fno-strict-aliasing -Wall -Wextra
+add_definitions(-std=gnu99 -fno-strict-aliasing -Wall -Wextra
     -Wchar-subscripts -Wundef -Wshadow -Wcast-align -Wwrite-strings
     -Wsign-compare -Wunused -Wno-unused-parameter -Wuninitialized -Winit-self
     -Wpointer-arith -Wformat-nonliteral
